Sources/RInAppMessaging/Router.swift

```diff
@@ -18,6 +18,7 @@ internal protocol RouterType: ErrorReportable {
     /// - Parameter cancelled: true when message display was cancelled
     func displayCampaign(_ campaign: Campaign,
                          associatedImageData: Data?,
+                         carouselImages: [UIImage?]?,
```

Comment:
could have been managed using different function or enum to identify what type of image going to display